### DISCUSSION TITLE
BuyNow

### DISCUSSION THREAD

**Discussion by JahidHasanCO** [JahidHasanCO/BuyNow-App#7] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/JahidHasanCO/BuyNow-App/releases/tag/1.0.0'>BuyNow</a>.</em>

**Comment by JahidHasanCO** [comment]
Nice

**Reply by tH3aim** [reply]
Hi, can we speak a little bit about the code? 💯

**Comment by tH3aim** [comment]
Hi, can we speak about the code a little bit?

**Reply by JahidHasanCO** [reply]
Of course

**Reply by tH3aim** [reply]
Thanks for the quick reply.  I found your app on GitHub and it looks really nice. The thing is that I have a class test on Thursday and I wanted to take your app. And I need some help with some code.

**Reply by tH3aim** [reply]
I am new to coding. Just for the record.

**Reply by JahidHasanCO** [reply]
https://www.facebook.com/hJahidHasan.jrC?mibextid=ZbWKwL

Dm me here
